src/AppBundle/Ldap/Normalizer.php

```diff
@@ -56,6 +56,7 @@ public function denormalizeUsers(array $users)
                 $user[$mapping['lastName']][0],
                 $this->getUserAttributeIfExists($user, 'displayName', ''),
                 $user[$mapping['loginName']][0],
+                $user[$mapping['avatarUrl']][0],
```

Comment:
Mag je aannemen dat dit attribute altijd aanwezig is in LDAP? Of een 'ifExists' check gebruiken?